samples/subsys/bindesc/hello_bindesc/README.rst

```diff
@@ -0,0 +1,26 @@
+.. _hello_bindesc-sample:
+
+hello_bindesc Sample Application
+################################
+
+Overview
+********
+
+A simple sample of binary descriptor definition and usage.
+
+Building and Running
+********************
+
+Follow these steps to build the `hello_bindesc` sample application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/bindesc/hello_bindesc
+   :board: <board to use>
+   :goals: build
+   :compact:
+
+To see all binary descriptors, run:
+
+.. code-block:: bash
+
+   west bindesc dump build/zephyr/zephyr.bin
```

Comment:
would be nice to have more details in the readme, at least the output of dump...

(note that when testing with native_posix, zephyr.bin does not exist)